extensions/graphql/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@loopback/graphql",
+  "version": "0.1.0",
+  "description": "LoopBack's graphql integration",
+  "engines": {
+    "node": ">=10"
+  },
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "acceptance": "lb-mocha \"dist/__tests__/acceptance/**/*.js\"",
+    "build": "lb-tsc --copy-resources",
+    "clean": "lb-clean loopback-graphql*.tgz dist *.tsbuildinfo package",
+    "pretest": "npm run build",
+    "integration": "lb-mocha \"dist/__tests__/integration/**/*.js\"",
+    "test": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "unit": "lb-mocha \"dist/__tests__/unit/**/*.js\"",
+    "demo": "npm run build && node dist/__examples__/graphql-test/src/index.js",
+    "verify": "npm pack && tar xf loopback-graphql*.tgz && tree package && npm run clean"
+  },
+  "author": "IBM Corp.",
+  "copyright.owner": "IBM Corp.",
+  "license": "MIT",
+  "dependencies": {
+    "@loopback/boot": "^2.4.0",
+    "@loopback/core": "^2.9.2",
+    "@loopback/http-server": "^2.1.10",
+    "@loopback/metadata": "^2.2.3",
```

Comment:
I think we should use `@loopback/core` instead `@loopback/metadata` for consistency with other extensions.